src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx

```diff
@@ -252,12 +253,16 @@ export default class ImportStep extends React.PureComponent<Props, State> {
     return { model, contents, type: isEmote ? ItemType.EMOTE : ItemType.WEARABLE }
   }
 
+  handleOpenMoreInformation = () => {
+    window.open('https://docs.decentraland.org/decentraland/creating-wearables/', '_blank', 'noopener noreferrer')
+  }
+
   renderMoreInformation() {
     return (
       <span>
         {t('create_single_item_modal.import_information', {
           link: (
-            <a href="https://docs.decentraland.org/decentraland/creating-wearables/" target="_blank" rel="noopener noreferrer">
+            <a href="javascript" onClick={preventDefault(this.handleOpenMoreInformation)}>
```

Comment:
could `<Button basic onClick={preventDefault(this.handleOpenMoreInformation)}>` works in this scenario?